src/components/dialogs/connectivity-edition.js

```diff
@@ -198,51 +216,136 @@ export const useConnectivityValue = ({
                 busOrBusbarSection: null,
                 connectionDirection: null,
                 connectionName: null,
+                connectionPosition: null,
             };
 
         const ret = {
             voltageLevel: ided(voltageLevelObjOrId),
             busOrBusbarSection: ided(busOrBusbarSectionObjOrId),
             connectionDirection: ided(connectionDirection),
             connectionName: ided(connectionName),
+            connectionPosition: ided(connectionPosition),
         };
         return ret;
     }, [
         voltageLevelObjOrId,
         busOrBusbarSectionObjOrId,
         connectionDirection,
         connectionName,
+        connectionPosition,
     ]);
 
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
     const render = useMemo(() => {
         return (
-            <Grid container direction={direction || 'row'} spacing={2}>
-                <Grid item xs={gridSize} align="start">
-                    {voltageLevelField}
-                </Grid>
-                <Grid item xs={gridSize} align="start">
-                    {busOrBusbarSectionField}
+            <>
+                <Grid container direction={direction || 'row'} spacing={2}>
+                    <Grid
+                        item
+                        xs={withPosition && withDirectionsInfos ? 4 : gridSize}
```

Comment:
Could this expression, `withPosition && withDirectionsInfos ? 4 : gridSize` be factorized in a const ?